lib/exceptions.rb

```diff
@@ -38,4 +38,17 @@ def message
       "Well, that's weird."
     end
   end
+
+  class MergedEntityError < StandardError
+    attr_reader :merged_entity
+
+    def initialize(merged_entity)
```

Comment:
One minor point here to think about. The default way exceptions work in ruby is that the first argument to ` .new ` is provide an optional different message than the default:

![excpetions_divide_by_zero](https://user-images.githubusercontent.com/8505044/33838775-8667730a-de5e-11e7-9da5-a3ecb4843857.png)

This changes that convention and will cause a srange error if we forget that later:
![merged_entity](https://user-images.githubusercontent.com/8505044/33838916-cca54658-de5e-11e7-9e78-2b5654ee38c7.png)

I don't know that it's worth changing necessarily because it's unlikely we're going to be raising this exception all that much, but I thought i'd put down the comment somewhere...